keyboards/cannonkeys/cerberus/keyboard.json

```
@@ -0,0 +1,755 @@
+{
+    "manufacturer": "CannonKeys",
+    "keyboard_name": "Cerberus",
+    "maintainer": "awkannan",
+    "backlight": {
+        "breathing": true,
+        "levels": 6,
+        "pin": "A6"
+    },
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "backlight": true,
+        "bootmagic": true,
+        "command": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "F1",
+        "on_state": 0
+    },
+    "matrix_pins": {
+        "cols": ["B0", "C15", "C14", "A7", "A5", "C13", "A4", "A2", "A1", "A0", "B9", "B8", "B7", "A3", "B6", "B4", "B3", "A15"],
+        "rows": ["B11", "B10", "B2", "F0", "B5"]
+    },
+    "processor": "STM32F072",
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 4
+    },
+    "url": "https://cannonkeys.com",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0031",
+        "vid": "0xCA04"
+    },
+    "ws2812": {
+        "driver": "spi",
+        "pin": "B15"
+    },
+    "community_layouts": ["tkl_nofrow_ansi", "tkl_nofrow_iso"],
+    "layout_aliases": {
+        "LAYOUT_all": "LAYOUT"
+    },
+    "layouts": {
+        "LAYOUT": {
```

layout alias not required as renamed `LAYOUT` to `LAYOUT_all`
```suggestion
    "layouts": {
        "LAYOUT_all": {
```